packages/recommend-react/src/FrequentlyBoughtTogether.tsx

```diff
@@ -14,10 +15,10 @@ const UncontrolledFrequentlyBoughtTogether = createFrequentlyBoughtTogetherCompo
   }
 );
 
-export type FrequentlyBoughtTogetherProps<
-  TObject
-> = GetFrequentlyBoughtTogetherProps<TObject> &
-  Omit<
+export type FrequentlyBoughtTogetherProps<TObject> = Omit<
+  GetFrequentlyBoughtTogetherProps<TObject>,
+  'recommendClient'
+> & { recommendClient?: RecommendClient } & Omit<
```

Comment:
as this is used multiple times, you can make a utility:

```ts
type OptionalRecommendClient<T> = Omit<T, 'recommendClient'> & { recommendClient?: T['recommendClient'] }
```

Reply:
✅ : https://github.com/algolia/recommend/pull/131/commits/2af99ace34588ed75b07db4bc3ef619872a9751e